### ISSUE TITLE
App installation and permission issues

### ISSUE THREAD

**Issue by shizukilovaton-notabot** [dominostars/playtranslate#2]
The app will not install if GooglePlay Protect is enabled.
After disabling GooglePlay Protect app can be installed, however the permission for accessibility cannot be granted, the option is grayed out.
I was still able to grant permissions with the help of shizuku and app ops.
Also, the floating icon disappears every time I open PPSSPP (as soon as I close or minimize the app, the floating icon appears again). 

I'm using Android 14
Snapdragon 695
8gb ram
Stock Motorola ROM
No root.

<img width="720" height="720" alt="Image" src="https://github.com/user-attachments/assets/64fe9b5d-9efe-4389-966e-10620a4492d4" />

**Comment by dominostars** [comment]
Hey! Sorry I totally missed this, thanks for posting. For the issue with disabled accessibility settings, you can do this on-device with these steps:

Open Settings → Apps → PlayTranslate
Tap the ⋮ menu (top right)
Tap Allow restricted settings
Authenticate when prompted

I added a section to the readme (as well as to the onboarding flow) to help with this. 

As for your other issues, I will investigate. Thanks again

**Comment by dominostars** [comment]
Are you still having your issue with PPSSPP? If so, when you open PlayTranslate after the floating icon disappears, does it prompt you to send crash logs? If so, could you send those? If not, could you still try sending the logs via the "Export Logs" button towards the bottom of settings in the support section?

**Comment by FillerBunnyNinjaShark** [comment]
There's no dots in the top right hand corner on my Retroid Pocket Mini. There's no way to enable the app

**Comment by dominostars** [comment]
FYI v2.2.0 will support "Display Over Other Apps" with MediaProjection, which won't perform as well as accessibility mode, but will bypass these issues on your devices. Coming soon

**Comment by dominostars** [comment]
v2.2.0 is released with support of media projection and using "Display over other apps", which sidesteps this issue. Auto translations does not work as well as accessibility, but you can upgrade later if you find away around permission restrictions
